readme.md

```diff
@@ -119,15 +151,20 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+> the bang (!) modifies the @artist object its called on.
 
 ## Question 10
 
 Using jQuery, write an AJAX request to `http://tunr.com/artists` that would create a new artist with the name of 'Resin Laying Deer Figurine, Gold', and pop up a box saying "All done!" when complete.
 
 ```js
-// Your code starts here...
-
+// $(ajax).on("click", function(){
+  type: 'artist',
+  url: "http://tunr.com/artists"
+  name: "Resin Laying Deer Figurine, Gold"
+  });
+  alert("All done!");
+});
 ```
 
```

Comment:
A few things...

1. You can place an AJAX call inside of an event listener, but you do not chain an event listener directly to `$.ajax`.

2. An AJAX call begins with `$.ajax`, not $(ajax).

3. When making a `POST` request with AJAX, we (a) need to indicate that it is a `POST` request using the `method` property and (b) pass in data as an **object** using the `data` property.

4. When we want something to happen after an AJAX call has run, we place that code inside of a promise like `.done`, `.fail` or `.then`.

So the final AJAX call should look something like this...

```js
$.ajax({
  method: "POST",
  url: "http://tunr.com/artists",
  dataType: "json",
  data: {
    name: "Resin Laying Deer Figurine, Gold"
  }
}).done(function(){
  alert("All done!");
});
```